Makefile

```diff
@@ -32,10 +33,15 @@ cross: clean
 .PHONY: vendor
 vendor: vendor.conf
 	@vndr 2> /dev/null
-	@script/validate/check-git-diff vendor
+	@scripts/validate/check-git-diff vendor
+
+## Generate man pages from go source and markdown
+.PHONY: manpages
+manpages:
+	@man/generate.sh
```

Comment:
This doesn't need to be `.PHONY`, this is one of the rare cases when a Makefile is actually appropriate.

I believe something like this would be correct:

```
man/man*: man/
    @man/generate.sh

manpages: man/man*
```